### PR TITLE
Add hitachi HNAS driver to manila barclamp

### PR DESCRIPTION
**Pull request by Itxaka**
Adds migration, schema+json for new attributes,
haml template for new attributes, driver to manila.conf,
locale entries for new attributes

### CODE REVIEW

File: crowbar_framework/config/locales/manila/en.yml [hunk 1]
```diff
@@ -58,6 +60,16 @@ en:
...
+            hitachi:
+              hds_hnas_cluster_admin_ip0: 'The IP of the clusters admin node. Only set in HNAS multinode clusters.'
+              hds_hnas_evs_id: 'Specify which EVS this backend is assigned to.'
+              hds_hnas_evs_ip: 'Specify IP for mounting shares.'
+              hds_hnas_file_system_name: 'Specify file-system name for creating shares.'
+              hds_hnas_ip: 'HNAS management interface IP for communication between Manila controller and HNAS.'
+              hds_hnas_password: 'HNAS user password. Required only if private key is not provided.'
+              hds_hnas_ssh_private_key: 'RSA/DSA private key value used to connect into HNAS. Required only if password is not provided.'
+              hds_hnas_stalled_job_timeout: 'The time (in seconds) to wait for stalled HNAS jobs before aborting.'
+              hds_hnas_user: 'HNAS username Base64 String in order to perform tasks such as create file-systems and network interfaces.'
```
Comment (vuntz) [inline crowbar_framework/config/locales/manila/en.yml]: I think the strings need to be reworked to be more consistent with the other strings (see the NetApp ones). They're generally shorter, don't contain a verb and don't add with a dot.

Comment (Itxaka) [inline crowbar_framework/config/locales/manila/en.yml]: Same as in the other PR, as I have no idea I just copied what it says on the openstack config page for the driver :=)

**Comment by toabctl** [comment]
@vuntz Do we want to support this driver now? And do we have hardware to test it?

**Comment by vuntz** [comment]
> @vuntz Do we want to support this driver now? And do we have hardware to test it?

Yes and not yet :-)

**Comment by vuntz** [comment]
Similarly to https://github.com/crowbar/crowbar-openstack/pull/464: +1 for merging it and iterating over the UI strings later on.

**Comment by Itxaka** [comment]
rebased :clap:

**Comment by nicolasbock** [comment]
+1

**Comment by vuntz** [comment]
+1. @Itxaka feel free to remove the "do not merge yet" tag and merge :-)

**Pull request merged**